plugins/hls-explicit-imports-plugin/src/Ide/Plugin/ExplicitImports.hs

```diff
@@ -71,33 +87,36 @@ descriptorForModules recorder pred plId =
   (defaultPluginDescriptor plId)
     {
       -- This plugin provides a command handler
-      pluginCommands = [importLensCommand],
+      pluginCommands = [PluginCommand importCommandId "Explicit import command" runImportCommand],
       -- This plugin defines a new rule
-      pluginRules = minimalImportsRule recorder,
-      pluginHandlers = mconcat
-        [ -- This plugin provides code lenses
-          mkPluginHandler SMethod_TextDocumentCodeLens $ lensProvider pred
+      pluginRules = minimalImportsRule recorder pred,
+      pluginHandlers =
+         -- This plugin provides code lenses
+           mkPluginHandler SMethod_TextDocumentCodeLens (lensProvider pred)
+        <> mkPluginHandler SMethod_CodeLensResolve lensResolveProvider
```

Comment:
Do we want to design a similar paired handler function for lenses like we have for code actions?

Reply:
Since the client never advertises a code lens resolve capability, we have to assume that all clients support it, and can't change our behaviour depending on whether they support it.

Reply:
seems busted, I opened https://github.com/microsoft/language-server-protocol/issues/1761

Reply:
I think it's very likely that almost no clients support it so this may just break for many clients.

Reply:
I think that actually not having a capability for it must mean that all clients (need to?) support it. It's also a realtively important resolve type, because you send all your codelens every edit. Vscode works with resolve, and I know the rust server uses it too.

Reply:
Could still be useful to have the paired function since maybe we can simplify some stuff that's always the same like the data handling.